presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -43,17 +40,23 @@
 @Test(singleThreaded = true)
 public class TestJdbcQueryBuilder
 {
-    private IDBI dbi;
-    private Handle dummyHandle;
+    private TestingDatabase database;
+    private String catalogName;
+    private JdbcClient jdbcClient;
+
+    // private IDBI dbi;
```

Comment:
We always remove code rather than commenting it out. Old code can be found in the Git history.